integration/helpers/instance.go

```diff
@@ -1056,12 +1058,12 @@ func (i *TeleInstance) StartProxy(cfg ProxyConfig) (reversetunnel.Server, error)
 		case service.ProxyReverseTunnelReady:
 			ts, ok := re.Payload.(reversetunnel.Server)
 			if ok {
-				return ts, nil
+				return ts, process, nil
 			}
 		}
 	}
 
-	return nil, nil
+	return nil, nil, nil
```

Comment:
Not sure if any test flow reaches this line but  if `process` seems to be available so why not return it ?   
I know that this is legacy code not related to this PR but returning `nil, nil, nil` looks really suspicious.

Reply:
I've made it an explicit error, as nothing good should happen if control even lands there.